docker-compose.jsonnet

```diff
@@ -13,14 +13,15 @@
 
   // For each service, whether to deploy that service.
   enabled:: {
```

Comment:
For master we want to keep the default components - downloader, restreamer, backfiller, cutter, thrimshim, ngnix and now thrimbletrimmer enabled while disabling others.

Reply:
Fixed that.